cmd/autoowners/main.go

```diff
@@ -363,14 +362,14 @@ func main() {
 	if err := secretAgent.Start([]string{o.githubToken}); err != nil {
 		logrus.WithError(err).Fatalf("Error starting secrets agent.")
 	}
-	gc = github.NewClient(secretAgent.GetTokenGenerator(o.githubToken), secretAgent.Censor, github.DefaultGraphQLEndpoint, github.DefaultAPIEndpoint)
+	gc := github.NewClient(secretAgent.GetTokenGenerator(o.githubToken), secretAgent.Censor, github.DefaultGraphQLEndpoint, github.DefaultAPIEndpoint)
```

Comment:
this comment review is not related to your changes in this PR, but it worths commenting. You don't have to `Start` any secretAgent since this tool is not going to be used as a daemon. Just create a closure that returns the bytes of your key and pass it to the `NewClient`. You can do something similar with the `Censor` too.

Reply:
I have to admit this is a copy/paste from autobumper at upstream.
Is `secretAgent`'s task to ensure the reload if secret changes?

Each run is about 30 mins for syncing owners for all the folders.
What about the `secret` changed during the run?

Another reason as you pointed out, is that with agent i get Censor for free.

Reply:
I might add a minimum secretAgent functionality in upstream's API for cases like this. I didn't also compare the resources that the binary will use with the secretAgent's Start and without.